public/graph.js

```diff
@@ -448,7 +452,7 @@ class GraphController {
                     "height",
                     d => y_total_received_sms_range(d[0]) - y_total_received_sms_range(d[1])
                 )
-                .attr("width", Width / Object.keys(dailyReceivedTotal).length);
+                .attr("width", (Width / Object.keys(dailyReceivedTotal).length) - innerPadding);
```

Comment:
It would nice to have the padding at the start of the day rather than at the end. This will help us prevent the first bar from covering the y-axis bar. The alternative is we fix that separately and only reduce the current inner padding.
![image](https://user-images.githubusercontent.com/23135322/80076987-91059d00-8555-11ea-82c6-87ad6c216903.png)

Reply:
Done